packages/charting/src/components/StackedBarChart/examples/MultiStackedBarChart.Example.tsx

```diff
@@ -0,0 +1,28 @@
+import * as React from 'react';
+import { MultiStackedBarChart } from '@uifabric/charting/lib/StackedBarChart';
+import { ILegendDataItem } from '@uifabric/charting/lib/components/Legend/Legend.types';
+
+export const MultiStackedBarChartExample: React.SFC<{}> = () => {
+  const points = [
+    [
+      { x: 'Debit card numbers (EU and USA)', y: 40 },
+      { x: 'Passport numbers (USA)', y: 23 },
+      { x: 'Social security numbers', y: 35 },
+      { x: 'Credit card numbers', y: 87 },
+      { x: 'Tax identification numbers (USA)', y: 87 }
+    ],
+    []
```

Comment:
No this will show multiple, this is an example to demonstrate the scenario where if we have no data in the second bar chart.

---
In reply to: [209397393](https://github.com/OfficeDev/office-ui-fabric-react/pull/5906#discussion_r209397393) [](ancestors = 209397393)